container/features/pom.xml

```diff
@@ -141,6 +141,7 @@
             <feature>atomikos</feature>
             <feature>batik</feature>
             <feature>c3p0</feature>
+            <feature>HikariCP</feature>
```

Comment:
Can you change this to hikari-cp to match the naming convention of other Karaf features? Thanks.